docs/s3.md

```diff
@@ -56,6 +56,33 @@ Based on our evaluations, however, we recommend that you use axial or combined t
 Axial topics tend to result in the most coherent topics, while angular topics result in the most distinct ones.
 The combined approach is a reasonable compromise between the two methods, and is thus the default.
 
+### Dynamic Topic Modeling *(Optional)*
+
+$S^3$ can also be used as a dynamic topic model.
+Temporally changing components are found using the following steps:
+
+1. Fit a global $S^3$ model over the whole corpus.
+2. Estimate unmixing matrix for each time-slice by fitting a linear regression from the embeddings in the time slice to the document-topic-matrix for the time slice estimated by the global model.
+3. Estimate term importances for each time slice the same way as the global model.
+
+```python
+from turftopic import SemanticSignalSeparation
+
+model = SemanticSignalSeparation(10).fit_dynamic(corpus, timestamps=ts, bins=10)
+model.plot_topics_over_time()
+```
+
+!!! info
+    Topics over time in $S^3$ are treated slightly differently to most other models.
+    This is because topics are not proportional in $S^3$, and can tip below zero.
+    In the timeslices where a topic is below zero, its **negative definition** is displayed.
+
+<figure>
+  <img src="../images/dynamic_s3.png" width="85%" style="margin-left: auto;margin-right: auto;">
+  <figcaption> Topics over time in a dynamic Semantic Signal Separation model. </figcaption>
+</figure>
```

Comment:
Is it interactive in the notebook? in which case it might be worth showing that using a gif

Reply:
It is interactive, but ideally it would be embedded as HTML then. I don't know how to do that in MKDocs, could have a look at it tho